core/src/main/scala/org/http4s/rho/bits/package.scala

```diff
@@ -0,0 +1,5 @@
+package org.http4s.rho
+
+package object bits {
```

Comment:
This is a strange place for a public API.

Reply:
I wasn't intending for 'RequestLike' to be public.  Its only used internally right now to deal with the difference between a `Request` and `AuthedRequest`